scripts/provisioning/s3_setup

```diff
@@ -48,6 +48,7 @@ where:
 --updateclusterid        Fetch and update cluster id in S3 config
 --cleanup                Cleanup S3 accounts created for backgroundelete.
 --createauthjkspassword  Setup random jks password for Auth server
+--validateprerequisites  Perform validation of the mini provisioning pre-requisites for S3 during post_install
```

Comment:
we are removing all options from s3_setup script and adding only few, like config, post_install, init etc.

Please refer to this PR: https://github.com/Seagate/cortx-s3server/pull/636

Reply:
To take this change, I'll need to wait for PR: #636 to be merged to mini-provisioner. This will cause the current task to spill over this sprint. Please let me know if this is ok. CC @AjinkyaDhumal 
Alternatively, I think we can address this later in a separate ticket or avoid changes in s3_setup in the current ticket.

Reply:
@AjinkyaDhumal  @amitwac2608 
Please review the updated changes following the discussion.